lib/index.ts

```diff
@@ -98,12 +98,16 @@ export async function inspect(
     const parseResult = parseTree(result, options.dev);
     const { javaVersion, mavenVersion } = parseVersions(versionResult);
     let callGraph: CallGraph | undefined;
+    let callGraphMetrics = {};
     if (options.reachableVulns) {
       debug(`getting call graph from path ${targetPath}`);
       try {
         callGraph = await javaCallGraphBuilder.getCallGraphMvn(
           path.dirname(targetPath),
         );
+        callGraphMetrics = {
+          callGraphMetrics: javaCallGraphBuilder.runtimeMetrics(),
```

Comment:
should it be mocked somewhere in tests?

Reply:
What about `callGraphMetrics = javaCallGraphBuilder.runtimeMetrics()` ?
And in line 128 `callGraphMetrics, `

Reply:
> What about `callGraphMetrics = javaCallGraphBuilder.runtimeMetrics()` ?
> And in line 128 `callGraphMetrics,`
This way `meta` will have an `undefined` `callGraphMetrics` when `reachableVulns` is off. I choose to do the way I did to avoid having this key when `reachableVulns` was not selected.



> should it be mocked somewhere in tests?
Not necessarily, the function is a noop, I don't see the need of mocking it just to check if the function was called.

Reply:
Added a test :)

Reply:
Got it! seems reasonable, maybe naming would have me less confused?
`callGraphOptionalMetrics` as the outside variable